### PR TITLE
Add name and version fields to package.json in webpack guide

### PR DESCRIPTION
**Pull request by hackebrot**
Hey there! 👋 

I've run into issues when trying to run ``$ lektor server -f webpack``. It kept complaining about not being able to find ``name`` and ``version`` in package.json.

I've added these fields to my config and seemed to resolve the problem.

Please let me know your thoughts! Thank you for making Lektor! 👍

### CODE REVIEW

**Comment by singingwolfboy** [comment]
Hi @hackebrot, thanks for the pull request! (And sorry it took so long for someone to review it...)

Rather than providing specific values for the `package.json` file, I think it would be better to recommend that users run [`npm init`](https://docs.npmjs.com/cli/init). This will interactively prompt them for the values to create a `package.json` file, and provide reasonable defaults as well. Can you modify this pull request to suggest `npm init`?

**Comment by hackebrot** [comment]
Hi @singingwolfboy! 😄 

I've updated the PR. Please let me know your thoughts!

File: content/docs/guides/webpack/contents.lr [hunk 1]
```diff
@@ -34,12 +34,29 @@ Now you need to configure webpack.  The plugin expects a webpack project in the
...
+
+[npm-init :ext](https://docs.npmjs.com/cli/init) is a command-line tool to
+interactively create a ``package.json`` file.
+
+```
+$ npm init
+```
+
+This will ask you a bunch of questions (invoke with ``--yes`` to use default
+values) and then generate the file for you:
```
Comment (singingwolfboy) [inline content/docs/guides/webpack/contents.lr]: Can you indicate that the provided JSON file is just an example, and shouldn't be copy-pasted verbatim?

File: content/docs/guides/webpack/contents.lr [hunk 2]
```diff
@@ -34,12 +34,29 @@ Now you need to configure webpack.  The plugin expects a webpack project in the
 
 ### `package.json`
 
-This file instructs `npm` which packages we will need.  All we need for a
-start is to create an almost empty file:
+This file instructs `npm` which packages we will need.
+
+[npm-init :ext](https://docs.npmjs.com/cli/init) is a command-line tool to
+interactively create a ``package.json`` file.
```
Comment (singingwolfboy) [inline content/docs/guides/webpack/contents.lr]: Can you specifically say that you need to _run_ `npm init`?

Comment (hackebrot) [inline content/docs/guides/webpack/contents.lr]: Not quite sure, I follow. ``npm init`` is mentioned on line 43

File: content/docs/guides/webpack/contents.lr (hunk 1 above)
Comment (hackebrot) [inline content/docs/guides/webpack/contents.lr]: Sure!

**Comment by hackebrot** [comment]
Thank you for the review and merging this @singingwolfboy! 🙇

**Pull request merged**